src/Sulu/Bundle/AdminBundle/Metadata/FormMetadata/TagMetadata.php

```diff
@@ -42,4 +47,30 @@ public function setPriority(?int $priority): void
     {
         $this->priority = $priority;
     }
+
+    public function setAttributes(array $attributes): void
+    {
+        $this->attributes = $attributes;
+    }
+
+    public function getAttributes()
+    {
+        return $this->attributes;
+    }
+
+    public function getAttribute(string $name)
+    {
+        return $this->attributes[$name] ?? null;
+    }
+
+    public function hasAttributes(array $attributes): bool
+    {
+        foreach ($this->attributes as $key => $value) {
+            if (($attributes[$key] ?? null) !== $value) {
```

Comment:
Are you sure that's correct? It's not really easy to understand, and I have encountered an error. If I put something like this on the `PageAdmin`, it will also show pages that have this tag without the given attribute:

```php
->addMetadataRequestParameters([
    'tags[sulu_page.type][type]' => 'test',
])
```

This will also return template that have the `sulu_page.type` tag without any attribute.